features/support/api_helpers.rb

```diff
@@ -52,15 +56,15 @@ def ensure_user_exists(user_details)
   creation_response = call_api(:post, "/users", payload: {
     users: user_details,
     updated_by: "functional tests",
-  })
+  }, safe_for_smoke_tests: true)
```

Comment:
Appears to only be used for the `Given /^I have a production ([a-z-]+) user` step. I didn't realise we actually had user accounts for production smoke tests... but if so then I guess it's OK? Feel's a little weird as you have to assume creating user accounts on production is always an OK thing to do as any new feature could use this step